app/api/resources/resolver_view.py

```diff
@@ -1,49 +1,96 @@
+import os
 from fastapi import APIRouter, Request
 from typing import Optional
-from fastapi.responses import RedirectResponse
+from fastapi.responses import RedirectResponse, HTMLResponse
 from loguru import logger
 import requests, logging
+import aiohttp
+from dotenv import load_dotenv
+from jinja2 import Environment, FileSystemLoader
+from jinja2 import Template
```

Comment:
Why two lines (9, 10)?

Reply:
Removing, as I am not using it anywhere.